internal/rbd/rbd_attach.go

```diff
@@ -313,6 +321,9 @@ func appendRbdNbdCliOptions(cmdArgs []string, userOptions string) []string {
 	if !strings.Contains(userOptions, setNbdIOTimeout) {
 		cmdArgs = append(cmdArgs, fmt.Sprintf("--%s=%d", setNbdIOTimeout, defaultNbdIOTimeout))
 	}
+	if cookie != "" && hasNBDCookieSupport {
```

Comment:
prevent this duplication by adding a `appendRbdNbdCookie()` helper?

Reply:
Same as above.